app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+  def index
+    @queried_food = params[:q]
+    @count_total_results = FoodFacade.total_hits(params[:q])
+    @found_foods = FoodFacade.search_food(params[:q])[0..9]
```

Comment:
Both of these methods call to the same service (api call), follow up in my comment in your food_facade to discuss a different approach to this, as to avoid making the call twice.